engine/tests/mechanics/games/test_credentials_fragments.py

```
@@ -0,0 +1,132 @@
+        # Two credentials games (e.g. a scheduled + a randomized shift) must not
+        # collide on a shared global fragment uid in the client registry.
+        game_a, handler_a = _game()
+        game_b, handler_b = _game()
+        handler_a.receive_move(game_a, ("inspect", "passport"))
+        handler_b.receive_move(game_b, ("inspect", "passport"))
+
+        def candidate_uid(handler, game):
+            return [
+                p for p in _by_type(handler.get_journal_fragments(game), PieceFragment)
+                if p.kind == "candidate"
+            ][0].uid
```

Ruff (0.15.14)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)